packages/core/echo/echo-db/src/core-db/core-database.ts

```diff
@@ -189,8 +189,16 @@ export class CoreDatabase {
   }
 
   // TODO(Mykola): Reconcile with `getObjectById`.
-  async loadObjectCoreById(objectId: string, { timeout }: { timeout?: number } = {}): Promise<ObjectCore | undefined> {
+  async loadObjectCoreById(
```

Comment:
doesn't seem right that this is becoming a part of a public API. It's ok if this is just temporary to get the release out, but in that case I'd add a TODO to remove expectedDocumentId option